modules/ROOT/assets/attachments/sg.yaml

```
@@ -638,10 +641,17 @@ properties:
               document.cookie = cookie1String;
               ```
             default: 'SyncGatewaySession'
+
+          sgreplicate_enabled:
+            type: boolean
+            default: true -- replication is enabled on this database
```

```suggestion
            default: true -- this Sync Gateway node can be assigned sg-replicate replications for this database
```
Probably best to avoid the generic term 'replication', as that would also describe Couchbase Lite replications.